docs/contribute/packages/releasing.rst

```diff
@@ -0,0 +1,77 @@
+Release Guidelines
+------------------
+
+Versioning
+==========
+
+Pyfar uses the `Semantic Versioning <https://semver.org>`_ scheme.
+This means that version numbers are of the form ``MAJOR.MINOR.PATCH``.
+
+In short, this means the following:
+
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backward compatible manner
+- PATCH version when you make backward compatible bug fixes
+
+If unsure, please refer to the `Semantic Versioning`_ documentation for details.
+
+PRs falling into the PATCH category are made against the main branch (and consequently need a branch off main),
+while MINOR and MAJOR changes are made against the develop branch (and consequently need a branch off develop).
+Note that PRs that only change documentation or examples are considered PATCH changes.
```

Comment:
I would suggest to mention something like this also at the semantic versioning bulletpoint in pr_workflow.rst, rather than only linking this info. This would ensure branching off the correct branch.